README.md

```diff
@@ -267,6 +267,9 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
 });
 ```
 
+**If you set the authorizer to authorizer:application** you need to create an `authorizers/application.js` inside your application (app/authorizers/applications.js). See the following section on implementing a custom authorizer.
+
```

Comment:
I think for now the most important thing is to get the [How does it work?](https://github.com/simplabs/ember-simple-auth/blob/jj-abrams/README.md#how-does-it-work) and [How do I use it](https://github.com/simplabs/ember-simple-auth/blob/jj-abrams/README.md#how-do-i-use-it) sections right. Would be great if you could review them.

Especially the [How do I use it](https://github.com/simplabs/ember-simple-auth/blob/jj-abrams/README.md#how-do-i-use-it) section should really cover the steps 80% of the users would have to take to set up a fairly standard integration which it currently doesn't really.